algoliasearch/index_test.go

```diff
@@ -544,6 +544,21 @@ func TestIndexingAndSearch(t *testing.T) {
 		}
 	}
 
+	t.Log("TestIndexAndSearch: Search for \"elon musk\" with \"(clickAnalytics:true)\" parameter")
```

Comment:
Could you rename the prefix into `TestIndexingAndSearch` to match the other logs and the testing function name.